SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -39,6 +83,27 @@ void UGlobalStateManager::ApplyUpdate(const Worker_ComponentUpdate& Update)
 	}
 }
 
+void UGlobalStateManager::ApplyMapUpdate(const Worker_ComponentUpdate& Update)
```

Comment:
`ApplyMapURLUpdate`? I got a bit confused if this was a schema map update or something else.

Reply:
Name updated to `ApplyDeploymentMapURLUpdate` since Mike also requested the `Deployment` inclusion.